### DISCUSSION TITLE
Shortcut for search

### DISCUSSION THREAD

**Discussion by hkjels** [community/community#6881] [Other Feature Feedback, Questions, & Ideas]
For the last couple of weeks, the global shortcut for search `/` has not worked. I'm using `Safari 15.1`
_Not sure if this is the right place to report, so I'm sorry if it's not_

**Comment by gaknoll** [comment]
Thanks for reporting this @hkjels - this is the right place! I'm not able to reproduce this but I've filed an issue for our team to look into this further.

**Reply by hkjels** [reply]
I doubt that it's related, but I also have this error-line in the console when opening the site
`Unrecognized Content-Security-Policy directive 'worker-src'.`
